prow/plugins/plugins.go

```diff
@@ -514,6 +515,25 @@ func (pa *PluginAgent) Config() *Configuration {
 	return pa.configuration
 }
 
+// ensureConfig ensure that the configuration
+// doesn't contain any nil value
+func ensureConfig(config *Configuration) error {
+	var errors []string
+	for repo, configuration := range config.Plugins {
+		for _, plugin := range configuration {
+			if plugin == "heart" && config.Heart == nil {
+				org := strings.Split(repo, "/")[0]
+				errors = append(errors, fmt.Sprintf("unexpected nil config for plugin %s for %s and %s", plugin, repo, org))
+			}
+		}
+	}
+
+	if len(errors) > 0 {
+		return fmt.Errorf("empty configurations:\n\t%v", strings.Join(errors, "\n\t"))
```

Comment:
Can you use `utilerrors.NewAggregate` here?

Reply:
Yep ! `NewAggregate` need errors as parameters i'll make the changes